cmd/view_cluster_manifest.go

```diff
@@ -0,0 +1,153 @@
+// Copyright © 2022-2023 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package cmd
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+	"unicode"
+
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/obolnetwork/charon/app/errors"
+)
+
+func newViewClusterManifestCmd(runFunc func(io.Writer, string) error) *cobra.Command {
+	var manifestFilePath string
+
+	cmd := &cobra.Command{
+		Use:   "view-cluster-manifest",
+		Short: "Shows cluster manifest contents",
+		Long:  `Opens and shows the specified cluster manifest by printing its content in JSON form.`,
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runFunc(cmd.OutOrStdout(), manifestFilePath)
+		},
+	}
+
+	cmd.Flags().StringVar(&manifestFilePath, "manifest-file", "cluster-manifest.pb", "The path to the cluster manifest file.")
+
+	return cmd
+}
+
+func runViewClusterManifest(out io.Writer, manifestFilePath string) error {
+	return viewClusterManifest(manifestFilePath, out)
+}
+
+func viewClusterManifest(manifestFilePath string, out io.Writer) error {
```

Comment:
nit: add godoc

Reply:
I removed that function and replaced it with a plain `runViewClusterManifest` - I don't think this function needs godoc (isn't exported, meaning of the function is clear enough).